documentation/meta/media_properties/etls.md

```diff
@@ -0,0 +1,218 @@
+# Extract-Transform-Load flows
+
+The diagrams below show how data flows through Openverse, and the various
+transformation steps that might happen at each stage.
+
+**Guide**:
+
+- **Blue boxes** represent places where data sits at rest.
+- **Purple boxes** represent transformation steps. Each box links to a longer
+  description below.
+
+## Current flow
+
+```{mermaid}
+flowchart TD
+
+%% Graph definitions
+    subgraph AF [Airflow]
+    P{{Provider}} --> M(MediaStore):::ETL
+    M --> T{{TSV}}
+    T --> IC1(Index constraints):::ETL
+    IC1 --> C[(Catalog DB)]
+    C --> B(Batched update):::ETL --> C
+    end
+
+    subgraph IS [Ingestion Server]
+    C --> ING(Deleted media removal):::ETL
+    ING --> TT[(Temp Table)]
+    TT --> CF(Cleaning & filtering):::ETL --> TT
+    TT --> A[(API DB)]
+    A --> IC2(Index constraints):::ETL --> A
+    A --> ESF(Index creation & deleted media filtering):::ETL
+    ESF --> ES[(Elasticsearch)]
+    end
+
+    A --> DL(Dead link filtering):::ETL
+    DL --> CL[Client]
```

Comment:
```
    DL --> CL[Client]
```

Obviously from a computing and data flow standpoint this name makes sense, but we don't refer to the API as a "client" anywhere else, really. Is there a way we could expand this (maybe a parenthetical including "Django API") so that others can "connect the dots" between this graphic and our stack as understood by developers?

Reply:
Ah, see I meant for "Client" here to be someone _using_ the API (which could be the Gutenberg integration, a WP plugin, etc). I'll try to clarify that!

Reply:
I actually don't have the Django API on here anywhere, and I think I should! I'll expand on it a bit more 😄